be/src/exec/pipeline/scan/olap_chunk_source.cpp

```diff
@@ -126,27 +99,6 @@ void OlapChunkSource::_init_counter(RuntimeState* state) {
     _io_timer = ADD_TIMER(_runtime_profile, "IOTime");
 }
 
-Status OlapChunkSource::_build_scan_range(RuntimeState* state) {
-    RETURN_IF_ERROR(_conjuncts_manager.get_key_ranges(&_key_ranges));
-    _conjuncts_manager.get_not_push_down_conjuncts(&_not_push_down_conjuncts);
-    _dict_optimize_parser.rewrite_conjuncts<false>(&_not_push_down_conjuncts, state);
-
-    // FixMe(kks): Ensure this logic is right.
```

Comment:
@kangkaisen Could you ensure it or remove the comment?

Reply:
The logic should be right and the comment could be removed.